public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
```

Comment:
A global CSS is tricky. We tried something like this with https://github.com/hoodiehq/hoodie-css, but it turned out to be a maintainance nightmare.

Let’s think about design guidelines and shared CSS later, I’d like to merge your PR and then iterate